src/components/RadioButton/RadioButton.ts

```diff
@@ -9,30 +9,59 @@ namespace fabric {
    * Adds basic demonstration functionality to .ms-RadioButton components.
    *
    */
-  export class RadioButton extends CheckBox {
+  export class RadioButton {
```

Comment:
Yeah I like the idea of having a base class, ChoiceField. RadioButton extending CheckBox may be dry but requiring checkbox every time you want to use radiobutton seemed like an unnecessary coupling. ChoiceField could also hold common styles between the two as well, like how it was originally implemented. We should hold off from reviewing this, looks like there needs to be some exploration / extra work done here to implement ChoiceField.